internal/http/imagerunner.go

```diff
@@ -169,6 +174,11 @@ func (c *ImageRunner) GetLogs(ctx context.Context, id string) (string, error) {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode == http.StatusNotFound {
+		return "", imagerunner.LogURLNotFoundError{
```

Comment:
Do we need this distinction with regards to the already existing `ErrResourceNotFound`?

Reply:
I wanted a more accurate error message, but didn't realize I could just do `err == ErrResourceNotFound`. I thought I had to do `errors.Is` or something to typecast.

Going with `ErrResourceNotFound` and wrapping it with a more contextual message.